lib/scripts/getDevDeps.js

```diff
@@ -0,0 +1,16 @@
+const execSync = require('child_process').execSync
+
+/**
+ * check for deployment environment variable ROOSEVELT_DEPLOYMENT
+ * if it exists don't install roosevelt devdeps
+ * otherwise, install devdeps
+ * @function installDevDeps
+ */
+;(function installDevDeps () {
```

Comment:
This doesn't need to be a self-executing function. In fact it doesn't need to live inside a function at all since it's encapsulated in this script file anyway.